src/components/wallet/ConnectGate.tsx

```
@@ -0,0 +1,54 @@
+          e.stopPropagation()
+        }
+      }}
+    >
+      <VisuallyHidden aria-live="polite">
+        Wallet not connected
+      </VisuallyHidden>
+      <ConnectButton
+        overridechainid={overrideChainId}
+        // Use default styled variant to match app header theme
+        // size and width align with primary CTAs
+        children={fallbackLabel || "Connect wallet to continue"}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)